sympy/integrals/__init__.py

```diff
@@ -9,7 +9,11 @@
     >>> integrate(sin(x),x)
     -cos(x)
 """
+from .deltafunctions import deltaintegrate
+from .heurisch import heurisch, heurisch_wrapper
 from .integrals import integrate, Integral, line_integrate
+from .manualintegrate import manualintegrate, integral_steps
+from .rationaltools import ratint
```

Comment:
Am 09.03.2015 um 16:41 schrieb Sergey B Kirpichev:
>>   from .integrals import integrate, Integral, line_integrate
>> +from .manualintegrate import manualintegrate, integral_steps
>> +from .rationaltools import ratint
>
> I'm assuming you should revert introduced exports in the __init__.py.

Understood.
What I do not understand is why; the docs clearly state that these 
symbols should be in, in this particular package. Either they should be 
removed from the docs (which I can do), or they should be added to the 
module (which is the current state of the patch); I have no particular 
preference either way, but I'd like to know why you think the docs are 
irrelevant in this case.

> But please stop talking.

Please be clearer.